_gsocproposals/2018/proposal_DIANAHEPanalysisfunctions.md

```diff
@@ -0,0 +1,46 @@
+---
+title: DIANA-HEP/analysisfunctions - Translate HEP analysis functions to modern paradigms
+layout: gsoc_proposal
+project: DIANA-HEP
+year: 2018
+organization:
+  - DIANA-HEP
```

Comment:
Hi Jim,
Looks good, just one last comment: you are specifying here DIANA-HEP as both project and organization. I would suggest to remove it from the organization list and just leave Princeton.

Reply:
Ok, done.